DepositeCalcTests/Tests/SettingsPageTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using DepositeCalcTests.Pages;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class SettingsPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Settings";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        public void LogoutLinkTest()
+        {
+            // Arrange
+            var settingsPage = new SettingsPage(driver);
+            var loginPage = new LoginPage(driver);
+
+            // Act
+            settingsPage.Logout();
+
+            // Assert
+            Assert.IsTrue(loginPage.IsOpened(), "Incorrect page");
+        }
+
+        [Test]
+        public void CancelButtonTest()
+        {
+            // Arrange
+            var settingsPage = new SettingsPage(driver);
+            string expectedUrl = "https://localhost:5001/Calculator";
+
+            // Act
+            settingsPage.Cancel();
+
+            // Assert
+            Assert.AreEqual(expectedUrl, driver.Url, "Incorrect page");
+        }
+
+        [TestCase("dd/MM/yyyy", "2024", "February", "29", "02", "29")]
+        [TestCase("dd-MM-yyyy", "2026", "March", "1", "03", "01")]
+        public void DateFormatStrartWithDayTest(string expectedDateFormat, string year, string month, string day, 
```

Comment:
This test can be written similar to 'NumberFormatTest': testcase parameters are dateFormat and expectedEndDate. Year, month, day and other values can be hard-codded in the test.

Also, you can remove the next test: DateFormatStrartWithMonthTest